Docs/ExtensionRestActions.txt

```diff
@@ -0,0 +1,97 @@
+# File comapible with https://sequencediagram.org/
```

Comment:
I'm not sure if we should commit the Docs file to the repo

Reply:
I figure it would help someone else edit the svg...